e2e/testcases/cli_test.go

```diff
@@ -833,3 +845,192 @@ func TestNomosVetNamespaceRepo(t *testing.T) {
 		})
 	}
 }
+
+func TestCLIBugreportNomosRunningCorrectly(t *testing.T) {
+	var bugReportZipName, bugReportDirName string
+	tmpDir := nomostest.TestDir(t)
+	tw := nomostesting.New(t, nomostesting.NomosCLI)
+	opts := nomostest.NewOptStruct(nomostest.TestClusterName(tw), tmpDir, tw)
+	nomostest.FreshTestEnv(t, opts)
+
+	// get bugreport
+	out, err := exec.Command("nomos", "bugreport").CombinedOutput()
+	if err != nil {
+		tw.Log(string(out))
+		tw.Fatal(err)
+	}
+
+	// locate zip file
+	bugReportZipName, err = getBugReportZipName("./", tw)
+	if err != nil {
+		tw.Fatal(err)
+	}
+
+	// move zip file to nt.tmpDir and unzip
+	tw.Log(fmt.Sprintf("move %s to %s", bugReportZipName, tmpDir))
+	_, err = exec.Command("mv", bugReportZipName, path.Join(tmpDir, bugReportZipName)).CombinedOutput()
+	if err != nil {
+		tw.Fatal(err)
+	}
```

Comment:
You can get rid of the move command by setting the working directory when running the `nomos bugreport` command:
```
cmd := exec.Command("nomos", "bugreport")
cmd.Dir = nt.TmpDir
out, err := cmd.Output()
```

Reply:
Done, setting work directory when running the `nomos bugreport`